src/concrete_minkowski_sum.jl

```diff
@@ -0,0 +1,190 @@
+```
+This is seen by noting that ``P ⊕ Q`` corresponds to the set of points
+``x ∈ \\mathbb{R}^n`` such that ``x = y + z`` with ``Ay ≤ b`` and ``Cz ≤ d``,
+hence it follows that ``Ay ≤ b`` and ``C(x-y) ≤ d``, and the inequality displayed
+above follows by considering the ``2n``-dimensional space ``\\binom{x}{y}``.
+The reduction from ``2n`` to ``n`` variables is performed using an elimination
+algorithm as described next.
+
+The elimination of variables depends on the concrete polyhedra library `Polyhedra`,
+which itself uses `CDDLib` for variable elimination. The available algorithms are:
+
+- `Polyhedra.FourierMotzkin`   -- computation of the projection by computing the
+                                  H-representation and applying the Fourier-Motzkin
+                                  elimination algorithm to it
+
+- `Polyhedra.BlockElimination` -- computation of the projection by computing the
+                                  H-representation and applying the block elimination
+                                  algorithm to it
+
+- `Polyhedra.ProjectGenerators` -- computation of the projection by computing the
+                                   V-representation and projecting them
+
+[1] Kvasnica, Michal. "Minkowski addition of convex polytopes." (2005): 1-10.
+"""
+function minkowski_sum(P::HPolyhedron{N}, Q::HPolyhedron{N};
+                       backend=nothing,
+                       algorithm=nothing,
+                       prune=true) where {N<:Real}
+
+    require(:Polyhedra; fun_name="minkowski_sum")
+    require(:CDDLib; fun_name="minkowski_sum")
+
+    A, b = tosimplehrep(P)
+    C, d = tosimplehrep(Q)
+    res = _minkowski_sum_hrep(A, b, C, d, backend=backend, algorithm=algorithm, prune=prune)
+    return res
+end
+
+function minkowski_sum(P::HPolytope{N}, Q::HPolytope{N};
+                       backend=nothing,
+                       algorithm=nothing,
+                       prune=true) where {N<:Real}
+
+    require(:Polyhedra; fun_name="minkowski_sum")
+    require(:CDDLib; fun_name="minkowski_sum")
+
+    A, b = tosimplehrep(P)
+    C, d = tosimplehrep(Q)
+    res = _minkowski_sum_hrep(A, b, C, d, backend=backend,
+                              algorithm=algorithm, prune=prune)
+    return convert(HPolytope, res)
+end
+
+# This function computes the concrete Minkowski sum between two polyhedra in
+# simple H-representation,
+# P = {x : Ax <= b} and Q = {x : Cx <= d}
+# using the projection methods. See the documentation of `minkowski_sum` for details.
+function _minkowski_sum_hrep(A::AbstractMatrix{N}, b::AbstractVector{N},
+                             C::AbstractMatrix{N}, d::AbstractVector{N};
+                             backend=nothing,
+                             algorithm=nothing,
+                             prune=true) where {N<:Real}
+
+    if backend == nothing
+        if N <: Rational
+            backend = CDDLib.Library(:Exact)
+        else
+            backend = CDDLib.Library()
```

Comment:
Maybe we can add an issue to outsource the backend choice to a helper function using dispatch, as we do in `init_Polyhedra.jl`.

Reply:
Ok. I did it in a new commit by dispatching in `N` with `default_cddlib_backend`.